palisade-service/README.md

```diff
@@ -18,8 +18,15 @@ limitations under the License.
 
 # Palisade Service
 
-The Palisade service is the entry point into the rest of the Palisade services, it accepts a request from the client and passes it forwards to the User Service to continue on the processing of the request. The client then receives a token used to connect
-to the Filtered Resource Service and retrieve the processed request.
+The Palisade Service is the entry point into the rest of the Palisade Services, it accepts a request from the client and passes it forwards to the User Service to continue on the processing of the request. 
```

Comment:
Suggest giving the reader a clear appreciation that this service's job it is to accept requests from the client and to forward onto a chain several services.  User service can still be mentioned as the next destination, but it is the overall chain of services is what is important.  Something that will get the reader appreciating that the token is in the immediate response from the RESTFul Palisade Service web service, but the processing/preparing of the  request is going on it the background.    Could keep it short and make a reference/link back to the project's main readme.

Reply:
updated